presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -208,36 +209,23 @@ private static void runConsole(QueryRunner queryRunner, ClientSession session, A
                             System.out.println(getHelpText());
                             continue;
                     }
+
+                    if (command.startsWith("source ")) {
+                        String path = command.split(" ")[1];
```

Comment:
Show it be allowed to give multiple paths (space separated)? 
In that case we can do something like:
~~~

foreach path in command.split
    query <- query_from_file(path)
    sql <- sql + query
execute(sql)